packages/react-conformance-make-styles/package.json

```diff
@@ -22,11 +22,11 @@
     "start": "yarn storybook"
   },
   "devDependencies": {
-    "@fluentui/eslint-plugin": "^1.4.1",
+    "@fluentui/eslint-plugin": "*",
     "@fluentui/scripts": "^1.0.0"
   },
   "dependencies": {
-    "@fluentui/react-conformance": "^0.5.0",
+    "@fluentui/react-conformance": "*",
```

Comment:
This is the one edge case. However #19804 will move this to devDependencies. Reasoning explained in that PR description